### PR TITLE
Added my info and pic to your website.

### PR DESCRIPTION
**Pull request by mishaalkandapath**
Please read and understand everything below
**Do not delete any text other than where you are instructed.**

**Students: If one of them is applicable to you. Please check it.**

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a Preview link and screenshot showning after and before the changes.
- [x] Images are `240 x 240` [w x h].
- [x] Included a description of change below.
- [ ] Squashed the commits.

# Changes done in this Pull Request
i added my student id to students.yml
Added my name,country,github username,website,facebook id.
I added my pic to images/students
- If your change will be reflected on the website, please provide a **Test-Link** (**Hint : `gh-pages`**)
- Fixes`#<Issue no.>`

### CODE REVIEW

File: _data/students.yml [hunk 1]
```diff
@@ -276,3 +276,12 @@
   lat: 26.9255
   lng: 75.7861
   
+- name: Mishaal Kandapath
+  country: Kuwait
+  github: ecomparer
+  image: mishaal_ecomparer.jpg
+  facebook: Mishaal Kandapath
+  website: ecomparer.cf
+
```
Comment (codedsun) [inline _data/students.yml]: Remove extra lines !

**Review by codedsun — CHANGES_REQUESTED** [review]

File: _data/students.yml (hunk 1 above)
Comment (mishaalkandapath) [reply _data/students.yml]: k

**Review by codedsun — APPROVED** [review]

**Review by codedsun — APPROVED** [review]

**Review by abishekvashok — APPROVED** [review]
LGTM, would be beter if you could squash the commits

**Review by nikit19 — APPROVED** [review]

File: _data/students.yml (hunk 1 above)
Comment (mishaalkandapath) [inline _data/students.yml]: i have done it

**Comment by mishaalkandapath** [comment]
hello anybody there?

**Comment by mishaalkandapath** [comment]
why is my pic and info not there in the official repo

**Pull request merged**